modules/tensor_mechanics/doc/content/source/materials/ADLAROMANCEStressUpdateBase.md

```diff
@@ -1,10 +1,9 @@
-# AD LAROM Creep Stress Update
-
-!syntax description /Materials/ADLAROMCreepStressUpdate
+# LAROMANCE Stress Update with Automatic Differentiation
 
```

Comment:
Probably ought to keep the `!syntax description` line, no?

Reply:
Learned something new here; since this is a base class, I don't register it using `registerADMooseObject`, so then the syntax description isn't available...I think.